lib/control_node.ex

```diff
@@ -2,17 +2,4 @@ defmodule ControlNode do
   @moduledoc """
   Documentation for `ControlNode`.
   """
-
-  @doc """
-  Hello world.
-
-  ## Examples
-
-      iex> ControlNode.hello()
-      :world
-
-  """
-  def hello do
-    :world
-  end
```

Comment:
Please remove the corresponding test too!

Reply:
> Please remove the corresponding test too!

@kansi Done.